server/src/controllers/Events.js

```diff
@@ -79,8 +82,11 @@ module.exports = {
     const eventId = req.params.id;
     const event = await Event.findById(eventId);
     await event.remove();
-    await cloudinary.v2.uploader.destroy(eventId);
-    res.status(204);
+    try {
+      await cloudinary.api.resource(eventId);
+      await cloudinary.v2.uploader.destroy(eventId);
+    } catch(error) {}
```

Comment:
It's better if send an error response here, presumably with a 500 response since something failed on the server side.